### PR TITLE
Close color picker when clicked outside of it

### PR DESCRIPTION
**Pull request by serhatyuna**
Implementation of #4

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -5,10 +5,25 @@ import './assets/main.css';
...
 import plausible from './plugins/plausible';
 
+const clickOutside = {
+  beforeMount: (el, binding) => {
+    el.clickOutsideEvent = (event) => {
+      if (!(el == event.target || el.contains(event.target))) {
+        binding.value();
+      }
+    };
+    document.addEventListener('click', el.clickOutsideEvent);
```
Comment (henripar) [inline src/main.js]: Can you change the `click` event to `mousedown`? That way the color picker wont close if the user first clicks inside the picker and then drags and releases the click outside of it. Thanks!

Comment (serhatyuna) [reply src/main.js]: Sure 💯

**Review by henripar — APPROVED** [review]
Perfect. Thank you!

**Comment by henripar** [comment]
closes #4

**Pull request merged**